extensions/3DTILES_layers/README.md

```diff
@@ -0,0 +1,98 @@
+# 3DTILES_layers
+
+## Overview
+
+This extension to 3D Tiles enables grouping of different types of content into layers. A layer is a user-defined grouping of content - for example, a tileset of a city may have a buildings layer, a roads layer and a vegetation layer. Each layer follows an independent layout for its tiles and there are not dependencies between layers. A layer may also associate with itself some metadata by conforming to a `class` declared in `3DTILES_metadata`.
+
+## Contributors
+
+Cesium
```

Comment:
@sanjeetsuhag @lilleyse I feel like we should talk about this briefly: since we have more extensions than we used to, how do we want to list contributors in each extension?